### DISCUSSION TITLE
Poll suggestions and ideas

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#14812] [Discussions]
The new option of "Polls" in GitHub discussions is really nice to have.

With this discussion would I now like to mention a few ideas that came into mind for me. Note that I so far hadn't time to test out this feature in its entirety so some stuff may already exist and I simply didn't know...

### Linking to Polls

(This may already be a thing)
It could be a nice addition to allow linking to polls similar to how you link to issues, pull requests and discussions, by using `#:id` as the pattern.

A nice addition to this could be to perhaps display the Poll itself, alongside available answers when the link itself is posted in a single line as the only content.

So for example would
```markdown
Hey there!

We're currently having a Poll open that is answering this topic and we would like your input on it.

#1234
```

Render the poll itself. If the poll is "closed" (Given that this is actually possible?) would it show the results instead.

### Timed Polls

One key feature pretty much every poll site usually has is a way to limit the poll to a certain timeframe, be it by setting a delay, or by defining a due date for it.
The Polls here should offer the same feature, as it would allow to collect information from the very moment and not have to worry about closing it yourself later on to avoid the results being altered.

### Multiple Choice

Sometimes, you want to allow people to provide multiple responses in a poll.
Imagine wanting to improve how you provide updates to the community and more precisely about what. Having just the option of a single answer per user would then be very limiting as a user may not just want to have updates about topic A but also about topic B, but can't select that answer too.

That's why Polls should have an option to set a max amount of answers a user can select (Defaults to 1). That way can you make some multiple-choice Polls for those situations.

**Comment by gmourier** [comment]
100% supports those ideas. This would make polls an impactful tool for teams to integrate into processes with fewer effort thanks to some of the automation presented here.
